File: CSharp/6kyu/CreatePhoneNumber.cs
using System.Linq;
using System;

namespace CSharp._6Kyu
{
    public static partial class Kata6
    {
        public static string CreatePhoneNumber(int[] numbers)
        {
            var n = string.Join("", numbers);

            return String.Format("({0}) {1}-{2}", n.Substring(0, 3), n.Substring(3, 3), n.Substring(6));
        }

        public static string CreatePhoneNumber_2(int[] numbers)
        {
            return long.Parse(String.Concat(numbers)).ToString("(000) 000-0000");
        }
    }
}

File: JavaScript/6kyu/CreatePhoneNumber.js
function createPhoneNumber(numbers) {
  numbers = numbers.join("");
  return (
    "(" +
    numbers.substring(0, 3) +
    ") " +
    numbers.substring(3, 6) +
    "-" +
    numbers.substring(6)
  );
}

function createPhoneNumber_2(numbers) {
  var format = "(xxx) xxx-xxxx";

  for (let i = 0; i < numbers.length; i++) {
    format = format.replace("x", numbers[i]);
  }

  return format;
}

var resutl = createPhoneNumber_2([1, 2, 3, 4, 5, 6, 7, 8, 9, 0]);
console.log(resutl);
